### PR TITLE
R4R:Bech32,Genesis,Gov Params,Features/gov,Cli-client/upgrade

### PR DESCRIPTION
**Pull request by MrXJC**

### CODE REVIEW

File: docs/zh/features/governance.md [hunk 1]
```diff
@@ -1,255 +1,166 @@
...
 
-## 基本功能描述
+## Basic Function Description
 
-1. 文本提议的链上治理
-2. 参数修改提议的链上治理
-3. 软件升级提议的链上治理
+1. On-chain governance proposals on text
+2. On-chain governance proposals on parameter change
+3. On-chain governance proposals on software upgrade
```
[the remark this answers is not in the document]
Comment (MrXJC) [reply docs/zh/features/governance.md]: .....ok

File: docs/features/basic-concepts/bech32-prefix.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Bech32 on IRISnet
+
+Bech32 is a new Bitcoin address format proposed by Pieter Wuille and Greg Maxwell. Besides Bitcoin addresses, Bech32 can encode any short binary data. In the IRIS network, keys and addresses may refer to a number of different roles in the network like accounts, validators etc. The IRIS network is designed to use the Bech32 address format to provide robust integrity checks on data. The human readable part(HRP) makes it more efficient to read and the users could see error messages.
+
```
Comment (chengwenxi) [inline docs/features/basic-concepts/bech32-prefix.md]: Add a href about bech32?

File: docs/features/basic-concepts/bech32-prefix.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+|fva|   IRISnet Validator's Operator Address|
+|fvp|   IRISnet Validator's Operator Public Key|
+|fca|   IRISnet Consensus Address|
+|fcp|    IRISnet Consensus Public Key|
+
+## Encoding
+
+Not all interfaces to users IRISnet should be exposed as bech32 interfaces. Many address are still in hex or base64 encoded form.
+
+To covert between other binary reprsentation of addresses and keys, it is important to first apply the Amino enocoding process before bech32 encoding.
```
Comment (chengwenxi) [inline docs/features/basic-concepts/bech32-prefix.md]: What about `Amino`？

File: docs/zh/features/governance.md [hunk 4]
```diff
@@ -12,18 +12,9 @@
 
 1. 任何用户可以发起提议，并抵押一部分token，如果超过`min_deposit`,提议进入投票，否则留在抵押期。其他人可以对在抵押期的提议进行抵押token，如果提议的抵押token总和超过`min_deposit`,则进入投票期。但若提议在抵押期停留的出块数目超过`max_deposit_period`，则提议被关闭。
 2. 进入投票期的提议，只有验证人和委托人可以进行投票。如果委托人没投票，则他继承他委托的验证人的投票选项。如果委托人投票了，则覆盖他委托的验证人的投票选项。当提议到达`voting_perid`,统计投票结果。
-3. 具体提议投票逻辑细节见[CosmosSDK-Gov-spec](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/spec/governance/overview.md)
+3. 我们统计结果有参与度的限制，其他逻辑细节见[CosmosSDK-Gov-spec](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/spec/governance/overview.md)
```
Comment (chengwenxi) [inline docs/zh/features/governance.md]: It's better to refer to a version?

Comment (MrXJC) [reply docs/zh/features/governance.md]: Great！

**Pull request merged**